### ISSUE TITLE
Dark / corrupted images with 1.3.2

### ISSUE THREAD

**Issue by Blzut3** [ECWolfEngine/ECWolf#221]
**[Original report](https://bitbucket.org/ecwolf/ecwolf/issues/221) by Darren Mason (Bitbucket: [darrenmason](https://bitbucket.org/darrenmason), GitHub: [darrenmason](https://github.com/darrenmason)).**

----------------------------------------

I am experiencing darkened / corrupted or incorrect colors after the init screen that causes the game to be unplayable.

I reverted to my old copy of version 1.2.3 which solved the issue.

I have attached a screenshot of the issue, which occurs on both wolf3d and spear of destiny.

Additionally, is it possible to make older versions such as 1.2.3 available for download?  

I am on Windows 10 latest, using x64 version.

Thanks.

**Comment by Blzut3** [comment]
If you switch between 1.2.x and 1.3.x this will happen.  Just hit enter a few time to start the game then press F11 until the gamma is back to 1.

All previous versions of ECWolf are archived: http://maniacsvault.net/ecwolf/files/ecwolf/1.x/

**Comment by Blzut3** [comment]
**Original comment by Darren Mason (Bitbucket: [darrenmason](https://bitbucket.org/darrenmason), GitHub: [darrenmason](https://github.com/darrenmason)).**

----------------------------------------

I see now that the cfg is in Appdata and being carried over.  I got:

"ReadConfig: Reading the Configuration.
Configuration:94:8:Error: Expected 'Float Constant' but got 'Identifier' instead."

So I just deleted the cfg and now everything is working.

Thanks for the help.
